src/usecases/patient/search-patient.ts

```diff
@@ -9,21 +11,40 @@ import { ExternalDataFetchError } from '@/services/hapi-fhir.service';
 
 export const searchPatientUseCase = async (
   context: { repo: IServerConfigRepository },
-  data: { patientId: string },
+  data: { patientId: string; email: string },
 ): Promise<string> => {
-  const serviceConfig = (await context.repo.findMany({})).find((x) => x);
-  if (!serviceConfig) {
+  const serviceConfigs = await context.repo.findMany({});
+  if (!serviceConfigs.length) {
     throw new ExternalDataFetchError('Missing Config error.');
   }
 
-  const service: IHapiFhirService =
-    HapiFhirServiceFactory.getService(serviceConfig);
-  const result = await service.searchPatient<FhirSearchResult<FhirPatient>>(
-    data.patientId,
-  );
+  let result: FhirSearchResult<FhirPatient>;
+  for (const serviceConfig of serviceConfigs) {
+    try {
+      const service: IHapiFhirService =
+        HapiFhirServiceFactory.getService(serviceConfig);
```

Comment:
Minor: Is this intended to be on a new line? possibly the linter that moved this onto a new line

Reply:
yes, the linter moved it